examples/poyo/README.md

```
@@ -1,33 +1,37 @@
-# POYO 🧠
-Official codebase for POYO published at NeurIPS 2023
-[[project page]](https://poyo-brain.github.io/)
-[[arxiv]](https://arxiv.org/abs/2310.16046)
+# POYO Example
 
-### Installation
+A minimal training example for [POYO](https://poyo-brain.github.io/).
 
+Please check out [POYO's official code repository](https://github.com/nerdslab/poyo) for fully-featured and paper-reproducing code.
+
+### Setup
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)